web/src/services/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const $axios = axios.create({
+  baseURL: '/.netlify/functions',
+  timeout: 10000, // 10 seconds
+});
+
+function getLocations(successCallback, errorCallback) {
+  // TODO: Add some caching logic here
```

Comment:
Can we add a follow-up ticket or note for this one?

Reply:
done!